sdk/eventhub/Azure.Messaging.EventHubs/stress/src/Configurations/TestConfiguration.cs

```diff
@@ -45,4 +45,46 @@ internal class TestConfiguration
     /// </summary>
     ///
     public int DurationInHours = 150;
+
+    /// <summary>
+    ///   Gets the partition count of the Event Hub being used for this test run.
+    /// </summary>
+    ///
+    /// <returns>The number of partitions in the Event Hub associated with this test.</returns>
+    ///
+    public async Task<int> GetEventHubPartitionCountAsync()
+    {
+        int partitionCount;
+
+        if (string.IsNullOrEmpty(EventHubsConnectionString) || string.IsNullOrEmpty(EventHub))
+        {
+            return 0;
+        }
+
+        await using (var producerClient = new EventHubProducerClient(EventHubsConnectionString, EventHub))
```

Comment:
Should we consider removing this method?   The value can be derived from the set of partition identifiers pulled by the other method.